server.js

```diff
@@ -29,7 +30,60 @@ app.post('/api/v1/diary', (request, response) => {
   } else {
     response.status(201).json({name})
   }
+})
+
+app.get('/api/v1/foods/:id', (request, response) => {
+  let id = request.params.id
+  Food.find(id)
+  .then( (data) => {
+    if (data.rowCount === 0) { return response.sendStatus(404) }
+
+    response.json(data.rows[0])
+  })
+})
 
+app.get('/api/v1/foods', (request, response) => {
+  Food.all()
+  .then( (data) => {
+    if (data.rowCount === 0) { return response.sendStatus(404) }
```

Comment:
If no foods are in the database, I think it's better to receive an empty array than returning a 404. Since 404 means the resource is not found, I think that is misleading. Empty and non-existent are different. 👍